types/audit.d.ts

```diff
@@ -70,17 +70,32 @@ declare global {
       wastedPercent?: number;
     }
 
+    export interface TapTargetTableItem extends DetailsObject {
```

Comment:
can this be moved to `tap-targets.js` as well? Since it's a `DetailsObject` and the table renderer can handle it, seems like it's ok to be internal to the audit until/unless we need it externally